spec/rubocop/ast/send_node_spec.rb

```diff
@@ -736,6 +736,126 @@ module Foo
     end
   end
 
+  describe '#nonmutating_binary_operator_method?' do
+    context 'with a nonmutating binary operator method' do
+      let(:source) { 'foo + bar' }
+
+      it { expect(send_node.nonmutating_binary_operator_method?).to be_truthy }
```

Comment:
Please use `be(true)` and `be(false)` until we can do better

Reply:
Yes, the whole test codebase is not written the way I would prefer, but I was consistent with what we have 😄